custom_model_runner/datarobot_drum/drum/drum.py

```diff
@@ -42,18 +45,50 @@
 
 import docker.errors
 
+from strictyaml import Bool, Int, load, Map, Optional, Str, YAMLError
+
 SERVER_PIPELINE = "prediction_server_pipeline.json.j2"
 PREDICTOR_PIPELINE = "prediction_pipeline.json.j2"
 
 
 class CMRunner:
     def __init__(self, runtime):
+        def _read_metadata(code_dir):
```

Comment:
I'd rather this not be an internal method...

Reply:
moved to common.py